go.mod

```diff
@@ -24,6 +24,7 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/munnerz/crd-schema-fuzz v1.0.0
 	github.com/onsi/ginkgo v1.16.5
```

Comment:
It seems suspicious that we are still importing v1. Is that expected? I was assuming this would not longer be present.

Reply:
Thank you. In `hack/bin/tools.go` there was still a reference to v1, fixed it.